controllers/route.go

```diff
@@ -265,6 +266,23 @@ func (as *AdminServer) Webhooks(w http.ResponseWriter, r *http.Request) {
 	getTemplate(w, "webhooks").ExecuteTemplate(w, "base", params)
 }
 
+// SuperLogin allows an admin to login to a user account without needing the password
+func (as *AdminServer) SuperLogin(w http.ResponseWriter, r *http.Request) {
+
+	if r.Method == "POST" {
+		username := r.FormValue("username")
+		u, err := models.GetUserByUsername(username)
+		if err != nil {
+			log.Error(err)
```

Comment:
A better pattern is to abort early. E.g. something like:

```go
if err != nil {
    log.Error(err)
    http.Error(w, http.StatusNotFound, err.Error())
    return
}
session := ctx....
```

Reply:
Good point, done.